src/utils/error-messages.ts

```diff
@@ -0,0 +1,34 @@
+enum Errors {
```

Comment:
I think this `enum` should be in error.ts file (where all things that relevant to error will be (like classes for exceptions))

It'll save you unnecessary exports like 
```typescript
import { StartGreaterThanTotalError } from '../utils/errors';
import Errors from '../utils/error-messages';
```

I think one import is nicer
```typescript
import { Errors, StartGreaterThanTotalError } from '../utils/errors';
```